controllers/manageddatabase_controller.go

```diff
@@ -177,13 +177,19 @@ func (c *ManagedDatabaseController) reconcileMigrationJob(oneMigration migration
 	for _, job := range jobsForDatabase.Items {
 		if job.Labels["migration-uid"] == string(oneMigration.version.UID) {
 			// This is the job for the migration in question
-			oneMigration.log.Info("Found matching migration")
+			oneMigration.log.Info("Found matching migration job")
 			foundJob = true
 
 			if job.Status.Succeeded > 0 {
-				oneMigration.log.Info("Migration is complete")
+				oneMigration.log.Info("Migration job is complete")
 
 				// TODO: should we write the metric here or wait until cleanup?
+			} else if job.Status.Active == 0 && job.Status.Conditions != nil {
```

Comment:
Can you break just this part out into a separate PR.

Reply:
https://github.com/quay/dba-operator/pull/9